app/services/MetricsService.scala

```diff
@@ -32,6 +32,9 @@ trait MetricsService {
   val getCustomerInfoTimer: Timer
   val getCustomerInfoCallFailureCounter: Counter
 
+  val getMandationStatusInfoTimer: Timer
```

Comment:
I asked Tom earlier if I needed to add Metrics for getting the mandation status, and he said it's not necessary. It may be worth having a quick conversation with him about this.